### ISSUE TITLE
Trying VSS action leads to %1 is not a valid Win32 application

### ISSUE THREAD

**Issue by audio441** [kopia/kopia#2440]
I'm trying to use the "Windows shadow copy" scripts at https://kopia.io/docs/advanced/actions/. 
I have the CLI and GUI installed on an Win 11 VM. 
I have minio running on an Ubuntu VM and have successfully stored snapshots and retrieved files.
When I set the before-snapshot-action, this happens:

PS C:\Users\admin> kopia repository connect from-config --file "C:\Users\admin\AppData\Roaming\kopia\repository.config" --enable-actions
Enter password to open repository:

Connected to repository.

NOTICE: Kopia will check for updates on GitHub every 7 days, starting 24 hours after first use.
To disable this behavior, set environment variable KOPIA_CHECK_FOR_UPDATES=false
Alternatively you can remove the file "C:\Users\admin\AppData\Roaming\kopia\repository.config.update-info.json".

PS C:\Users\admin> kopia policy show C:\Users
Policy for admin@desktop-4mi34db:C:\Users:

Retention:
  Annual snapshots:                     3   inherited from (global)
  Monthly snapshots:                   24   inherited from (global)
  Weekly snapshots:                     4   inherited from (global)
  Daily snapshots:                      7   inherited from (global)
  Hourly snapshots:                    48   inherited from (global)
  Latest snapshots:                    10   inherited from (global)
  Ignore identical snapshots:       false   inherited from (global)

Files policy:
  Ignore cache directories:          true   inherited from (global)
  No ignore rules:
  Read ignore rules from files:             inherited from (global)
    .kopiaignore
  Scan one filesystem only:         false   inherited from (global)

Error handling policy:
  Ignore file read errors:          false   inherited from (global)
  Ignore directory read errors:     false   inherited from (global)
  Ignore unknown types:              true   inherited from (global)

Scheduling policy:
  Scheduled snapshots:
  Snapshot times:                           inherited from (global)
    8:00
  Manual snapshot:                  false   inherited from (global)

Uploads:
  Max parallel snapshots (server/UI):   1   inherited from (global)
  Max parallel file reads:              -   inherited from (global)
  Parallel upload above size:       2 GiB   inherited from (global)

Compression disabled.

Run command before snapshot root:           (defined for this target)
  Command:
    C:\Users\admin\Documents\before.ps1
  Mode:                         essential
  Timeout:                           5m0s

Run command after snapshot root:            (defined for this target)
  Command:
    C:\Users\admin\Documents\after.ps1
  Mode:                         essential
  Timeout:                           5m0s

Logging details (0-none, 10-maximum):
  Directory snapshotted:                8   (defined for this target)
  Directory ignored:                    5   inherited from (global)
  Entry snapshotted:                    0   inherited from (global)
  Entry ignored:                        5   inherited from (global)
  Entry cache hit:                      0   inherited from (global)
  Entry cache miss:                     0   inherited from (global)

PS C:\Users\admin> kopia snapshot create C:\Users
Snapshotting admin@desktop-4mi34db:C:\Users ...
ERROR upload error: %1 is not a valid Win32 application.

I have tried some simple debugging, but it appears the before.ps1 is not getting executed. 
Any suggestions?

**Comment by Christmas-Cavalier** [comment]
It looks like you only have the path to the powershell script as the command. Call powershell.exe and give it the path to the script file instead. Example:

```
kopia policy set C:\Users --before-folder-action "powershell -WindowStyle Hidden C:\Users\admin\Documents\before.ps1"
```

**Comment by audio441** [comment]
Suggestion above did not help. I tried inspecting the code, but I'm not familiar with Go. I tried reinstalling with new version 12.1. But I still see under policy show C:/Users stuff that I tried before the reinstall. Also, I learned that directory policies are case sensitive: I have C:\Users and c:\users in show policy list. 

How (without a new windows install) can I completely start over? Just running the Windows installer for Kopia is not enough.
